### PR TITLE
Corrected Grammar Mistakes in README.md

### PR DESCRIPTION
**Pull request by coders-i**
I'm making some corrections to README.md file like Capitalization mistakes and some Grammar Mistakes.

### CODE REVIEW

**Comment by yliaog** [comment]
please sign cncf cla

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ python -m examples.example1
 
 ## Documentation
 
-All APIs and Models' documentation can be found at the [Generated client's README file](kubernetes/README.md)
+All APIs and Model's documentation can be found at the [Generated client's README file](kubernetes/README.md)
```
Comment (palnabarun) [inline README.md:75]: The previous version is actually the correct version. Please revert this.

File: README.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ Key:
...
 * `✗` No longer maintained; please upgrade.
 
-Kubernetes supports [three minor releases](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew) at a time. "Support" means we expect users to be running that version in production, though we may not port fixes back before the latest minor version. For example, when v1.3 comes out, v1.0 will no longer be supported. In consistent with Kubernetes support policy, we expect to support **three GA major releases** (corresponding to three Kubernetes minor releases) at a time.
+Kubernetes supports [three minor releases](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew) at a time. "Support" means we expect users to be running that version in production, though we may not port fixes back before the latest minor version. For example, when v1.3 comes out, v1.0 will no longer be supported. Inconsistent with the Kubernetes support policy, we expect to support **three GA major releases** (corresponding to three Kubernetes minor releases) at a time.
```
Comment (palnabarun) [inline README.md:129]: This change modifies the meaning of the sentence. We are consistent with the Kubernetes support policy.

Please revert this as well.

**Review by palnabarun — CHANGES_REQUESTED** [review]
Please find the requested changes inline.

Also, as @yliaog mentioned, you would need to sign the CLA to get the changes tested and merged.

**Comment by palnabarun** [comment]
@coders-i -- are you still working on this? If yes, can you please make the suggested changes and sign the Kubernetes CLA? If not, can we close this PR?

**Comment by coders-i** [comment]
Yes, Please close PR

*Priyanshu Arya*

**Pull request closed without merging**